implicit.cabal

```diff
@@ -49,7 +49,7 @@ Library
                 -Wall
                 -Wcompat
                 -Wmonomorphism-restriction
-                -Wmissing-local-signatures
```

Comment:
Changing this warning means that you don't have to put signatures on every variable declaration, only those at the top level of the file. This means you are still telling the compiler exactly what you want for every function in a module, exported or not, but you can let the type inference engine fill in the blanks for the local ones that aren't visible outside of a function.